### PR TITLE
Configuring JIT Provisioning Enhanced Feature

### PR DESCRIPTION
**Pull request by GANGANI**
## Purpose
Related feature improvement: https://github.com/wso2/product-is/issues/12017

### CODE REVIEW

File: en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md [hunk 1]
```diff
@@ -118,3 +118,20 @@ below:
...
+    enable_enhanced_feature = "true"
+    ```
+
+With the JIT provisioned enhanced feature following capabilities will be available.
+
+-   A Federated unique user id will be set as the username of the JIT provisioned user.
+-   Second-factor authenticators(Ex:- TOTP, Email OTP) will be able to configure for federated users when JIT provisioing is enabled
+-   When IDP deletes, all provisioned users from that IDP will be deleted.
+-   Editing JIT provisioned user’s attributes will not be allowed.
+-   JIT provisioned user accounts will be able to lock based on social identity.
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md]: Fixed https://github.com/wso2/docs-is/pull/2728/commits/5a3f6364dc388753086012f937879a4eec226164

File: en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md [hunk 2]
```diff
@@ -118,3 +118,20 @@ below:
...
+    ```toml
+    [authentication.jit_provisioning]
+    enable_enhanced_feature = "true"
+    ```
+
+With the JIT provisioned enhanced feature following capabilities will be available.
+
+-   A Federated unique user id will be set as the username of the JIT provisioned user.
+-   Second-factor authenticators(Ex:- TOTP, Email OTP) will be able to configure for federated users when JIT provisioing is enabled
+-   When IDP deletes, all provisioned users from that IDP will be deleted.
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md]: Fixed https://github.com/wso2/docs-is/pull/2728/commits/5a3f6364dc388753086012f937879a4eec226164

File: en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md [hunk 3]
```diff
@@ -118,3 +118,20 @@ below:
...
+### Configuring JIT Provisioning Enhanced Feature (Optional)
+
+Add the following configuration to deployment.toml in <IS_HOME>/conf directory.
+
+    ```toml
+    [authentication.jit_provisioning]
+    enable_enhanced_feature = "true"
+    ```
+
+With the JIT provisioned enhanced feature following capabilities will be available.
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md]: Fixed https://github.com/wso2/docs-is/pull/2728/commits/5a3f6364dc388753086012f937879a4eec226164

File: en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md [hunk 4]
```diff
@@ -118,3 +118,20 @@ below:
     provisioning user interfaces, see [Customizing Just-In-Time Provisioning
     User
     Interfaces](../../learn/customizing-just-in-time-provisioning-user-interfaces).
+    
+### Configuring JIT Provisioning Enhanced Feature (Optional)
+
+Add the following configuration to deployment.toml in <IS_HOME>/conf directory.
+
+    ```toml
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md]: Fixed https://github.com/wso2/docs-is/pull/2728/commits/5a3f6364dc388753086012f937879a4eec226164

File: en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md [hunk 5]
```diff
@@ -118,3 +118,20 @@ below:
     provisioning user interfaces, see [Customizing Just-In-Time Provisioning
     User
     Interfaces](../../learn/customizing-just-in-time-provisioning-user-interfaces).
+    
+### Configuring JIT Provisioning Enhanced Feature (Optional)
+
+Add the following configuration to deployment.toml in <IS_HOME>/conf directory.
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md]: Fixed https://github.com/wso2/docs-is/pull/2728/commits/5a3f6364dc388753086012f937879a4eec226164

**Review by gomathyk — APPROVED** [review]

**Pull request merged**